<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="4" passed="2">
  <reporter-output>
    <line>
      <![CDATA[Running First @test to check mandatory fields]]>
    </line>
    <line>
      <![CDATA[Navigating to the Contact Menu]]>
    </line>
    <line>
      <![CDATA[Incorrect details to validate the Error messages]]>
    </line>
    <line>
      <![CDATA[Running Second @test to check success submission]]>
    </line>
    <line>
      <![CDATA[Navigating to the Contact Menu]]>
    </line>
    <line>
      <![CDATA[Details for the user submitted]]>
    </line>
    <line>
      <![CDATA[Running third @test to check invalid field details]]>
    </line>
    <line>
      <![CDATA[Navigating to the Contact Menu]]>
    </line>
    <line>
      <![CDATA[Invalid details to validate the Error messages]]>
    </line>
    <line>
      <![CDATA[Running fourth @test to check success submission]]>
    </line>
    <line>
      <![CDATA[Navigating to the shop Menu]]>
    </line>
    <line>
      <![CDATA[Adding items to the cart]]>
    </line>
    <line>
      <![CDATA[Validating items in the cart]]>
    </line>
  </reporter-output>
  <suite name="planIT_TestSuite" duration-ms="78814" started-at="2021-05-23T13:01:00Z" finished-at="2021-05-23T13:02:19Z">
    <groups>
    </groups>
    <test name="PlanIT_Tests" duration-ms="78814" started-at="2021-05-23T13:01:00Z" finished-at="2021-05-23T13:02:19Z">
      <class name="planIT.tests.JupiterTests">
        <test-method status="PASS" signature="onStart()[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="onStart" is-config="true" duration-ms="1240" started-at="2021-05-23T23:00:59Z" finished-at="2021-05-23T23:01:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onStart -->
        <test-method status="PASS" signature="initialize()[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="initialize" is-config="true" duration-ms="150" started-at="2021-05-23T23:01:00Z" finished-at="2021-05-23T23:01:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="setup" is-config="true" duration-ms="7552" started-at="2021-05-23T23:01:00Z" finished-at="2021-05-23T23:01:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void planIT.tests.JupiterTests.mandatoryfieldserror()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@69653e16]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="mandatoryfieldserror()[pri:1, instance:planIT.tests.JupiterTests@64cd705f]" name="mandatoryfieldserror" duration-ms="15497" started-at="2021-05-23T23:01:08Z" finished-at="2021-05-23T23:01:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at planIT.pages.JupiterPages.validatecontact(JupiterPages.java:128)
	at planIT.tests.JupiterTests.mandatoryfieldserror(JupiterTests.java:58)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Running First @test to check mandatory fields]]>
            </line>
            <line>
              <![CDATA[Navigating to the Contact Menu]]>
            </line>
            <line>
              <![CDATA[Incorrect details to validate the Error messages]]>
            </line>
          </reporter-output>
        </test-method> <!-- mandatoryfieldserror -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="setup" is-config="true" duration-ms="1005" started-at="2021-05-23T23:01:23Z" finished-at="2021-05-23T23:01:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void planIT.tests.JupiterTests.successsubmission()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@69653e16]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="successsubmission()[pri:2, instance:planIT.tests.JupiterTests@64cd705f]" name="successsubmission" duration-ms="10210" started-at="2021-05-23T23:01:24Z" finished-at="2021-05-23T23:01:34Z">
          <reporter-output>
            <line>
              <![CDATA[Running Second @test to check success submission]]>
            </line>
            <line>
              <![CDATA[Navigating to the Contact Menu]]>
            </line>
            <line>
              <![CDATA[Details for the user submitted]]>
            </line>
          </reporter-output>
        </test-method> <!-- successsubmission -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="setup" is-config="true" duration-ms="989" started-at="2021-05-23T23:01:34Z" finished-at="2021-05-23T23:01:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void planIT.tests.JupiterTests.invaliddatavalidation()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@69653e16]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="invaliddatavalidation()[pri:3, instance:planIT.tests.JupiterTests@64cd705f]" name="invaliddatavalidation" duration-ms="4197" started-at="2021-05-23T23:01:35Z" finished-at="2021-05-23T23:01:40Z">
          <reporter-output>
            <line>
              <![CDATA[Running third @test to check invalid field details]]>
            </line>
            <line>
              <![CDATA[Navigating to the Contact Menu]]>
            </line>
            <line>
              <![CDATA[Invalid details to validate the Error messages]]>
            </line>
          </reporter-output>
        </test-method> <!-- invaliddatavalidation -->
        <test-method status="PASS" signature="setup(java.lang.reflect.Method, org.testng.ITestContext)[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="setup" is-config="true" duration-ms="118" started-at="2021-05-23T23:01:40Z" finished-at="2021-05-23T23:01:40Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void planIT.tests.JupiterTests.cartvalidation()]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[org.testng.TestRunner@69653e16]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="cartvalidation()[pri:4, instance:planIT.tests.JupiterTests@64cd705f]" name="cartvalidation" duration-ms="37825" started-at="2021-05-23T23:01:40Z" finished-at="2021-05-23T23:02:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at planIT.pages.JupiterPages.validatecart(JupiterPages.java:198)
	at planIT.tests.JupiterTests.cartvalidation(JupiterTests.java:128)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Running fourth @test to check success submission]]>
            </line>
            <line>
              <![CDATA[Navigating to the shop Menu]]>
            </line>
            <line>
              <![CDATA[Adding items to the cart]]>
            </line>
            <line>
              <![CDATA[Validating items in the cart]]>
            </line>
          </reporter-output>
        </test-method> <!-- cartvalidation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="tearDown" is-config="true" duration-ms="1246" started-at="2021-05-23T23:02:18Z" finished-at="2021-05-23T23:02:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="flushReports()[pri:0, instance:planIT.tests.JupiterTests@64cd705f]" name="flushReports" is-config="true" duration-ms="17" started-at="2021-05-23T23:02:19Z" finished-at="2021-05-23T23:02:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushReports -->
      </class> <!-- planIT.tests.JupiterTests -->
    </test> <!-- PlanIT_Tests -->
  </suite> <!-- planIT_TestSuite -->
</testng-results>
